### DISCUSSION TITLE
History View

### DISCUSSION THREAD

**Discussion by xeniode** [runew0lf/RuinedFooocus#181] [Q&A]
Is there still the history view that was in the original Fooocus? I could not find it.

And another tangent question. How can I save the images with metadata? I do not see a button. I can use the context menu to "Save IMage As..." in my browser, but OG Fooocus had a save button on the image itself.

Thanks in advance.

**Comment by yownas** [comment]
Hi. There is no history view, but the metadata is saved in all images and are saved in RuinedFooocus\outputs\ sorted by date.
Just clear out your prompt and drop the image in the main window to get the prompt (and all the other data)
